config/example/ir.yaml

```diff
@@ -124,11 +124,11 @@ contracts:
     zhivete: f584699bc2ff457d339fb09f16217042c1a42101 # Optional: override address of zhivete contract in sidechain
 
 profiler:
-  address: 127.0.0.1:6060  # Endpoint for application pprof profiling; disabled by default
+  address: localhost:6060  # Endpoint for application pprof profiling; disabled by default
   shutdown_timeout: 30s    # Timeout for profiling HTTP server graceful shutdown
 
 metrics:
-  address: 127.0.0.1:9090  # Endpoint for application prometheus metrics; disabled by default
+  address: localhost:9090  # Endpoint for application prometheus metrics; disabled by default
```

Comment:
What about adopting `prometheus` and `pprof` in IR, though?

Reply:
Fixed